File: WebApplication1/WebApplication1/DTO/Response/HallBookingDto.cs
using WebApplication1.Models;

namespace WebApplication1.DTO.Response
{
    public class HallBookingDto
    {
        public int? HallBookingId { get; set; }
        public DateTime FromDateTime { get; set; }
        public DateTime ToDateTime { get; set; }
        public double? Price { get; set; }
        public BookingStatus? Status { get; set; }
        public string  userEmail { get; set; }
        public string HallName { get; set; }
    }
}

File: angular-app/src/app/api/models/hall-booking-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BookingStatus } from './booking-status';
export interface HallBookingDto {
  fromDateTime?: string;
  hallBookingId?: null | number;
  hallName?: null | string;
  price?: null | number;
  status?: BookingStatus;
  toDateTime?: string;
  userEmail?: null | string;
}
